.travis.yml

```diff
@@ -9,22 +9,24 @@ env:
 matrix:
   fast_finish: true
   include:
-  - name: "py36-np111"
-    env: PY=3.6 NUMPY=1.11
+  - name: "py36-np116"
+    env: PY=3.6 NUMPY=1.16
   - name: "py36-np117"
-    env: PY=3.6 NUMPY=1.17
-  - name: "py38-np111"
-    env: PY=3.8 NUMPY=1.11
+    env: PY=3.6 NUMPY=1.19
```

Comment:
The name (numpy version) doesn't match the environment. And, apart from that, I'm also fine with dropping 3.6 entirely at this point.

Reply:
Argh! I checked those a thousand times and missed it anyway. Thanks!